Neos.ContentRepository.NodeAccess/Classes/FlowQueryOperations/IdOperation.php

```diff
@@ -21,6 +21,7 @@
 
 /**
  * Used to access the Node's identifier of a ContentRepository Node.
+ * @deprecated please use ${node.aggregateId} instead
```

Comment:
How do we deal with those deprecations – AFAIS the user won't notice them.
Wouldn't it make more sense to just remove the operation so they _do_?
Or what are the plans for a removal?

Reply:
haha thing is this was praised by us to BE THE way in beta10 https://discuss.neos.io/t/neos-9-beta-10-release/6578 and beta11 again https://discuss.neos.io/t/neos-9-beta-11-release/6618 so i dont know if we can just remove them again 😅 . This is a classic EIGENTOR. And for what i think we dont need to remove i before the beta as a dead fq doesnt hurt.